src/test/java/BaseTest.java

```diff
@@ -1,18 +1,39 @@
 import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
 import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.interactions.Actions;
 import org.openqa.selenium.safari.SafariDriver;
-import org.testng.annotations.BeforeMethod;
-import org.testng.annotations.BeforeSuite;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.*;
 
+import java.time.Duration;
 
 public class BaseTest {
+    public static WebDriver driver = null;
+    public static WebDriverWait wait;
+    public static Actions actions;
+
 
     @BeforeSuite
     static void setupClass() {
         WebDriverManager.chromedriver().setup();
     }
 
-}
+    @BeforeMethod
+    @Parameters({"BaseURL"})
+    public static void launchBrowser(String BaseURL) {
+        driver = new ChromeDriver();
+        actions = new Actions(driver);
```

Comment:
After the Chrome 111 update, you can no longer kick off a chromedriver instance unless you add an additional chrome option:
"--remote-allow-origins=*"

```
        ChromeOptions options = new ChromeOptions();
        options.addArguments("--disable-notifications","--remote-allow-origins=*", "--incognito","--start-maximized");
        driver = new ChromeDriver(options);
```


Below are the list of available and most commonly used arguments for ChromeOptions class

**start-maximized:** Opens Chrome in maximize mode
**incognito:** Opens Chrome in incognito mode
**headless:** Opens Chrome in headless mode
**disable-extensions:** Disables existing extensions on Chrome browser
**disable-popup-blocking:** Disables pop-ups displayed on Chrome browser
**make-default-browser:** Makes Chrome default browser
**version:** Prints chrome browser version
**disable-infobars:** Prevents Chrome from displaying the notification ‘Chrome is being controlled by automated software